elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
+        &lt;tagname>
```

Comment:
oh and yes i believe we're using &lt; to display the tag without it being rendered at HTML. i dont know about parsing it as a head tag. what is your thought behind the question?